File: app/src/main/java/com/bearcave/passageplanning/routes/ReadWaypoints.java
package com.bearcave.passageplanning.routes;

import com.bearcave.passageplanning.waypoints.database.Waypoint;

import java.util.List;

/**
 * Created by miwas on 24.05.17.
 */

public interface ReadWaypoints {
    List<Waypoint> readAllWaypoints();
}

File: app/src/main/java/com/bearcave/passageplanning/routes/ReadWaypoints.kt
package com.bearcave.passageplanning.routes

import com.bearcave.passageplanning.waypoints.database.Waypoint

/**
 * Created by miwas on 24.05.17.
 */

interface ReadWaypoints {
    fun readAllWaypoints(): List<Waypoint>
}
